packages/cli-dashboard/src/app.tsx

```diff
@@ -68,6 +69,11 @@ const App = () => {
   }, []);
 
   useEffect(() => {
+    sessionStorage.clear();
```

Comment:
What's the purpose of clearing `sessionStorage` here?

Reply:
Since the dashboard table's persistence settings are saved in sessionStorage.
Let's say the user analyses the site in `hi` and then in `en`, so if we don't clear storage the Hindi filters will stay in UI due to persistence.
So when the user refreshes the dashboard file in the browser, we remove the saved filters so we only show the latest language's filters.